### PR TITLE
Clean code-generator/generate-internal-groups.sh help info.

### PR DESCRIPTION
**Pull request by yanggangtony**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Clean the help info m
image
essage to match the whole key words.

#### Which issue(s) this PR fixes:
Fixes #

```release-note
None
```

### CODE REVIEW

**Comment by jiahuif** [comment]
/lgtm
/assign @deads2k 
Trivial cleanup, could you take a look and approve it?

File: staging/src/k8s.io/code-generator/generate-internal-groups.sh [hunk 1]
```diff
@@ -27,8 +27,8 @@ Usage: $(basename "$0") <generators> <output-package> <internal-apis-package> <e
 
   <generators>        the generators comma separated to run (deepcopy,defaulter,conversion,client,lister,informer,openapi) or "all".
   <output-package>    the output package name (e.g. github.com/example/project/pkg/generated).
-  <int-apis-package>  the internal types dir (e.g. github.com/example/project/pkg/apis).
-  <ext-apis-package>  the external types dir (e.g. github.com/example/project/pkg/apis or githubcom/example/apis).
+  <internal-apis-package>  the internal types dir (e.g. github.com/example/project/pkg/apis).
+  <extensiona-apis-package>  the external types dir (e.g. github.com/example/project/pkg/apis or githubcom/example/apis).
   <groups-versions>   the groups and their versions in the format "groupA:v1,v2 groupB:v1 groupC:v2", relative
                       to <api-package>.
   ...                 arbitrary flags passed to all generator binaries.
```
Comment (jiahuif) [inline staging/src/k8s.io/code-generator/generate-internal-groups.sh:34]:
```suggestion
  <generators>               the generators comma separated to run (deepcopy,defaulter,conversion,client,lister,informer,openapi) or "all".
  <output-package>           the output package name (e.g. github.com/example/project/pkg/generated).
  <internal-apis-package>    the internal types dir (e.g. github.com/example/project/pkg/apis).
  <extensiona-apis-package>  the external types dir (e.g. github.com/example/project/pkg/apis or githubcom/example/apis).
  <groups-versions>          the groups and their versions in the format "groupA:v1,v2 groupB:v1 groupC:v2", relative
                             to <api-package>.
  ...                        arbitrary flags passed to all generator binaries.
```

Comment (jiahuif) [reply staging/src/k8s.io/code-generator/generate-internal-groups.sh:34]: Add spaces to align the columns

**Review by jiahuif — CHANGES_REQUESTED** [review]

**Comment by jiahuif** [comment]
Could you also fix the typo "extensiona" in both places?

**Comment by yanggangtony** [comment]
@jiahuif 
Thanks for review.
Rebased.

**Comment by yanggangtony** [comment]
@deads2k 
Would you please take a look?

**Pull request closed without merging**